Competitive_Programming/C++/Matrix/Spiral_Matrix.md

```diff
@@ -0,0 +1,114 @@
+# Displaying a Matrix in Spiral Form 
+ In this question we will display a matrix in spiral form by traversing through 
+left-> right then 
+top-> bottom then 
+left->right then  
+bottom-> top and 
+so on  till we reach the center element.
+I have given a Example for you to get a better understanding:
+
+Given a matrix of size r\*c. Traverse the matrix in spiral form.
+
+## Example:
+
+ *Input*: r = 4, c = 4
+ 
+  matrix[][] = {{1, 2, 3, 4}, {5,6, 7, 8}, {9, 10, 11, 12}, {13, 14, 15,16}} 
+  
+  *Output*: 1 2 3 4 8 12 16 15 14 13 9 5 6 7 11 10
+***
+*Input*: r = 3, c = 4 
+
+ matrix[][] = {{1, 2, 3, 4}, {5, 6, 7, 8}, {9,10, 11, 12}}
+ 
+ *Output*: 1 2 3 4 8 12 11 10 9 5 6 7
```

Comment:
To make any Text/word bolder, you have to use 2*.
Above Input/Output words are not formatted to bold.